src/pybind/mgr/dashboard/tests/test_rbd_mirroring.py

```diff
@@ -48,7 +48,7 @@ class RbdMirroringSummaryControllerTest(ControllerTestCase):
     @classmethod
     def setup_server(cls):
         mgr.list_servers.return_value = mock_list_servers
-        mgr.get_metadata.return_value = mock_get_metadata
+        mgr.get_metadata = mock.Mock(return_value=mock_get_metadata)
```

Comment:
Is this change needed? As it seems unrelated.

Reply:
> Is this change needed?

I don't think so. But i looked more correct to me, so i added it. Will remove it again.

Reply:
Had to add it again, otherwise the unit tests will fail.